app/views/finance/ecf/contracts/show.html.erb

```diff
@@ -0,0 +1,3 @@
+<% content_for :before_content, govuk_back_link(text: "Back", href: finance_ecf_payment_breakdown_path(id: @ecf_lead_provider.id)) %>
```

Comment:
Do we not use I18n.t for these "Back" links?

Reply:
Slightly surprised that the BackLinkComponent doesn't default the text rather than fail if there's no text or content, but c'est la vie.

Reply:
At the moment there are 66 usages of `text: "Back"` in the version of code I am looking at 😂 

Maybe they call for some commonisation, but perhaps not in this PR :)